openupgrade_scripts/scripts/account/14.0.1.1/pre-migration.py

```diff
@@ -352,6 +343,17 @@ def add_move_id_field_account_payment(env):
         WHERE am.id = ap.move_id
         """,
     )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_move am
+        SET currency_id = ap.{}
+        FROM account_payment ap
+        WHERE am.id = ap.move_id
```

Comment:
```suggestion
        WHERE am.id = ap.move_id AND ap.{} is not null
```

Reply:
Unnecessary, `currency_id` is required on `account.payment` on Odoo 13

Reply:
Sorry, I meant

```suggestion
        WHERE am.id = ap.move_id AND am.currency_id != ap.{}
```

Reply:
Ok, done, thank you